deploy/crds/kieapp.crd.yaml

```diff
@@ -3820,6 +3823,10 @@ spec:
                               description: The image tag to use e.g. 7.9.0, this param
                                 is optional for custom image.
                               type: string
+                            jbpmCluster:
+                              description: JbpmCluster configuration for Kie Servers
```

Comment:
should this match the descrip in the form?
"Enable the clustering between kieserver, it could increase the number of kieservers"

Reply:
Aligned both with "Enable the KIE Server Jbpm clustering for processes fail-over, it could increase the number of kieservers"